features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -80,6 +93,12 @@ class Base64Serializer private constructor(
         this.asyncImageProcessingCallback = asyncImageProcessingCallback
     }
 
+    internal fun getDrawableScaledDimensions(
+        view: ImageView,
+        drawable: Drawable,
+        density: Float
+    ): Pair<Long, Long> = drawableUtils.getDrawableScaledDimensions(view, drawable, density)
```

Comment:
Don't we have any class for dimensions? just to be explicit.

Reply:
👍  I'll change this to use `android.util.Size`